.github/workflows/spack_build.yml

```diff
@@ -99,9 +99,9 @@ jobs:
     strategy:
       matrix:
         spack_spec:
-          - hiop@develop+mpi~raja~shared~kron~sparse ^openblas ^openmpi ^libevent~openssl
-          - hiop@develop~mpi~raja~shared~kron~sparse ^openblas ^libevent~openssl
-          - hiop@develop~mpi+raja~shared~kron~sparse ^openblas ^libevent~openssl
+          - hiop@develop+mpi~raja~shared~kron~sparse ^openmpi 
+          - hiop@develop~mpi~raja~shared~kron~sparse
```

Comment:
@nychiang this build is failing because for some reason it's trying to build `libflame`, and not `openblas`.

Looks at the concretization comparison between builds, and you'll see that a 3 hour build would make no sense for the smallest build of the batch here.

Reply:
Why would that need possibly need python for this build?

Reply:
python is required via
hiop -> amdlibflame -> aocl-utils -> doxygen -> graphviz -> gtkplus -> meson -> python-xxx
why do we need amdlibflame?  Concretization shows that it installs a lot of packages.
@cameronrutherford